pkgs/by-name/bi/bicpl/package.nix

```
@@ -9,14 +9,14 @@
 
 stdenv.mkDerivation {
   pname = "bicpl";
-  version = "unstable-2024-05-14";
+  version = "unstable-2024-10-24";
```

```suggestion
  version = "0-unstable-2024-10-24";
```
as per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning